guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -41,13 +41,30 @@ For information about supported workflows, see https://access.redhat.com/solutio
 endif::[]
 
 In {Project} provisioning, the PXE loader option defines the DHCP `filename` option to use during provisioning.
-For BIOS systems, use the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
-For UEFI systems, use the *PXEGrub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *PXEGrub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
-ifndef::satellite[]
-Use SecureBoot options to enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
-Other PXE loaders like PXELinux UEFI, Grub2 ELF or iPXE Chain, require additional configuration. These workflows are not documented at the moment.
+For BIOS systems, select the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
+For UEFI systems, select the *Grub2 UEFI* option to enable a TFTP client to download `grubx64.efi` file, or select the *Grub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` with the HTTP Boot feature.
```

Comment:
This is outside the scope of this PR. Just FYI. 

GRUB is an acronym. The UI labels should be "GRUB2 ..."